Lib/typing.py

```diff
@@ -1460,13 +1794,31 @@ def lookup_name(mapping: Mapping[KT, VT], key: KT, default: VT) -> VT:
     def __class_getitem__(cls, params):
         if not isinstance(params, tuple):
             params = (params,)
-        if not params and cls is not Tuple:
+
+        try:
+            num_class_params = len(cls.__parameters__)
+        except AttributeError:
+            only_class_parameter_is_typevartuple = False
+        else:
+            if (
+                    num_class_params == 1
+                    and isinstance(cls.__parameters__[0], TypeVarTuple)
+            ):
+                only_class_parameter_is_typevartuple = True
+            else:
+                only_class_parameter_is_typevartuple = False
```

Comment:
nit: replace with `only_class_parameter_is_typevartuple = <A and B>`

Reply:
Done.